MagicBytesValidator/Formats/Pdf.cs

```diff
@@ -12,11 +12,12 @@ public Pdf() : base(
         StartsWith([0x25, 0x50, 0x44, 0x46, 0x2D])
             .EndsWithAnyOf(
             [
-                [0x0A, 0x25, 0x25, 0x25, 0x45, 0x4F, 0x46],
+                [0x0A, 0x25, 0x25, 0x45, 0x4F, 0x46],
                 [0x0A, 0x25, 0x25, 0x45, 0x4F, 0x46, 0x0A],
+                [0x0A, 0x25, 0x25, 0x45, 0x4F, 0x46, 0x0A, 0x20],
                 [0x0D, 0x0A, 0x25, 0x25, 0x45, 0x4F, 0x46, 0x0D, 0x0A],
-                [0x0D, 0x25, 0x25, 0x45, 0x4F, 0x46, 0x0D],
-                [0x20, 0x0a, 0x25, 0x25, 0x45, 0x4f, 0x46, 0x0d, 0x0a]
+                [0x0D, 0x25, 0x25, 0x45, 0x4F, 0x46, 0x0D], /* looks strange, but garykessler says so. */
```

Comment:
All other sequences contain a `0x0A` except for this one. Weird, but this is exactly how it's printed on the website.